### PR TITLE
Dependencies made more visible

### PR DESCRIPTION
**Pull request by zekiahmetbayar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ its other dependencies to your GOPATH or GOBIN (usually `~/go/bin/`) with the
 go tool:
 
 ```console
+$ sudo apt install libpcap-dev
```
Comment (hwipl) [inline README.md]: This is a distribution-specific change.

**Comment by hwipl** [comment]
Hey, thanks for the PR :)

Improving the documentation is always good. But this would be a distribution-specific change.

What do you think about adding it as an example to the paragraph above? For example, changing

```listnd requires libpcap to capture packets from network interfaces. So, make sure you have libpcap installed.```

to something like this:

```listnd requires libpcap to capture packets from network interfaces. So, make sure you have libpcap installed (for example, on a debian-based distribution with `$ sudo apt install libpcap-dev`).```

File: README.md (hunk 1 above)
Comment (hwipl) [reply README.md]: See my other comment (https://github.com/hwipl/listnd/pull/2#issuecomment-1046643308)

**Comment by zekiahmetbayar** [comment]
Hi, thank you for your insightful feedback! I've updated the description again, trying to understand your request. I hope I was able to help you this time in presenting a more useful readme file to the community. 

If you want me to rearrange it, I'll gladly do it.  Healthy days!

**Comment by hwipl** [comment]
I think that's fine. Thanks for your contribution :)

**Pull request merged**